src/components/drop/Drop.tsx

```diff
@@ -1,34 +1,57 @@
+import * as classNames from 'classnames';
 import * as React from 'react';
 import * as ReactDOM from 'react-dom';
-import {DropPod} from './DropPod';
+import {IReactVaporState} from '../../ReactVapor';
+import {IDispatch, ReduxConnect} from '../../utils/ReduxUtils';
+import {DropPod, IDropPodProps} from './DropPod';
+import {DefaultGroups, DropActions} from './redux/DropActions';
+import {DropSelectors} from './redux/DropReducers';
 
-export interface IDropProps {
+export interface IDropOwnProps extends IDropPodProps {
+    id: string;
     renderOpenButton: (onClick: () => void) => React.ReactNode;
     buttonContainerProps?: React.HTMLProps<HTMLDivElement>;
     listContainerProps?: React.HTMLAttributes<HTMLDivElement>;
-    closeOnClick?: boolean;
-    positions?: string[];
-    selector?: string;
+    closeOnClickOutside?: boolean;
+    closeOnClickDrop?: boolean;
+    group?: string;

```

Comment:
de we need to add id or group is good to understand the principe of drop by group.

If not, I can add id to avoid confusion

Reply:
Its just by convention, everywhere else we use `id` so I think its best if we continue in that way to avoid confusion.
Also in the admin we already have a notion of groups and default groups so it would add in to the confusion.